src/node_perf.cc

```diff
@@ -336,14 +332,50 @@ void Init(Local<Object> target,
   Local<FunctionTemplate> pe = env->NewFunctionTemplate(PerformanceEntry::New);
   pe->InstanceTemplate()->SetInternalFieldCount(1);
   pe->SetClassName(performanceEntryString);
+
+  Local<Signature> signature = Signature::New(env->isolate(), pe);
+
+  Local<FunctionTemplate> get_performance_entry_name_templ =
+      FunctionTemplate::New(env->isolate(),
+                            GetPerformanceEntryName,
+                            Local<Value>(),
```

Comment:
I know they didn't use to get `env->as_external()`, so this should be fine. But for sake of consistency I'd rather still have `env->as_external()` here.

Reply:
Sure, that makes sense. Done 👍